scipy/stats/tests/test_multicomp.py

```diff
@@ -250,6 +250,89 @@ def test_alternatives(self, alternative, statistic, pvalue):
             )
             assert_allclose(res.pvalue, pvalue, atol=1e-7)
 
+    # filter warning that could happen during optimization
+    @pytest.mark.parametrize(
+        'alternative, allowance, ci_low, ci_high',
+        [
+            (
+                'two-sided', 11,
+                [0.7529028025053, -8.2470971974947, -15.2470971974947],
+                [21.2470971974947, 12.2470971974947, 5.2470971974947]
+             ),
+            (
+                'less', 9,
+                [2.4023682323149, -6.5976317676851, -13.5976317676851],
+                [np.inf, np.inf, np.inf]
+            ),
+            (
+                'greater', 9,
+                [-np.inf, -np.inf, -np.inf],
+                [19.5984402363662, 10.5984402363662, 3.5984402363662]
+            )
+        ]
+    )
+    def test_allowance(self, alternative, allowance, ci_low, ci_high):
+        # Example (a) from Dunnett1995 using R multcomp `glht`
```

Comment:
1955, maybe? (Probably needs to be changed elsewhere).

Reply:
Ark yes 👍